flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkScan.java

```diff
@@ -346,7 +346,7 @@ public void testPartitionTypes() throws Exception {
     assertRecords(run(), records, typesSchema);
   }
 
-  static void assertRecords(List<Row> results, List<Record> expectedRecords, Schema schema) {
+  public static void assertRecords(List<Row> results, List<Record> expectedRecords, Schema schema) {
```

Comment:
Maybe move this into a utility class now that it's public static?

Reply:
yes, I move it to `SimpleDataUtil`,this method can be used for the assertion of unsorted list，I have encountered similar needs in other PR

Reply:
We have something like it in the Hive tests as well, and I would guess Spark has one too 😄 
Do we have some other place even more accessible?

Reply:
@pvary  , Could you give me some cases in the Hive module? Let me see if we can move them to data or core module

Reply:
@zhangjun0x01: We have 2 flavors for checking the contents of the table:
- [HiveIcebergTestUtils.validateData(Table table, List<Record> expected, int sortBy)](https://github.com/apache/iceberg/blob/e4185cd18412711242d0233ff901fb185ce63c94/mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergTestUtils.java#L220)
- [HiveIcebergTestUtils.validateData(List<Record> expected, List<Record> actual, int sortBy)](https://github.com/apache/iceberg/blob/e4185cd18412711242d0233ff901fb185ce63c94/mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergTestUtils.java#L238)

Thanks,
Peter

Reply:
@pvary I search the spark code ,[ It does have a similar required](https://github.com/apache/iceberg/blob/master/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java#L512)， but  I think we should extract the  function in a new PR later ,what do you think?

Reply:
I am fine with moving this refactoring to another PR.
Thanks, Peter